public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -24,27 +24,41 @@ export interface Props {
   header?: string;
 }
 
+//string, unknown, string, boolean
+interface DisplayValue {
+  name: string;
+  value: unknown;
```

Comment:
should this type be unknown? do we have a type in grafana data for the union of possible value types from a field?